File: Api.Server/src/Services/Center/Login/Dto/WeChatClientLoginOutput.cs
namespace Fast.Center.Service.Login.Dto;

/// <summary>
/// <see cref="WeChatClientLoginOutput"/> 微信客户端登录输出
/// </summary>
public class WeChatClientLoginOutput
{
    /// <summary>
    /// 登录状态
    /// </summary>
    public LoginStatusEnum Status { get; set; }

    /// <summary>
    /// 消息
    /// </summary>
    public string Message { get; set; }

    /// <summary>
    /// 唯一用户标识
    /// </summary>
    public string OpenId { get; set; }

    /// <summary>
    /// 统一用户标识
    /// </summary>
    public string UnionId { get; set; }

    /// <summary>
    /// 手机
    /// </summary>
    public string Mobile { get; set; }

    /// <summary>
    /// 昵称
    /// </summary>
    public string NickName { get; set; }

    /// <summary>
    /// 头像
    /// </summary>
    public string Avatar { get; set; }
}

File: Web.Admin/src/api/services/Auth/login/models/WeChatClientLoginOutput.ts
import { LoginStatusEnum } from "@/api/enums/LoginStatusEnum";

/**
 * Fast.Center.Service.Login.Dto.WeChatClientLoginOutput 微信客户端登录输出
 */
export interface WeChatClientLoginOutput {
  /**
   * 
   */
  status?: LoginStatusEnum;
  /**
   * 消息
   */
  message?: string;
  /**
   * 唯一用户标识
   */
  openId?: string;
  /**
   * 统一用户标识
   */
  unionId?: string;
  /**
   * 手机
   */
  mobile?: string;
  /**
   * 昵称
   */
  nickName?: string;
  /**
   * 头像
   */
  avatar?: string;
}

